setup.cfg

```diff
@@ -47,9 +47,10 @@ install_requires =
     pyyaml>=4.2b1
     setuptools>=19.6
     sympy!=1.9,>=1.2  # see https://github.com/sympy/sympy/issues/22241
-    toml>=0.10.2
+    tomli-w>=0.4.0
     tqdm>=3.4.0
     unyt>=2.8.0
+    tomli>=1.2.3;python_version < '3.11'
```

Comment:
for the record, I picked the most recent versions of each package that had support for Python 3.6 as minimal, to make backporting to yt 4.0.x easier